spec/support/webmock.rb

```diff
@@ -0,0 +1 @@
+WebMock.disable_net_connect!
```

Comment:
`require 'webmock/spec'` will do this for you and slides conveniently in to `spec/spec_helper.rb`.